sources/Context.cpp

```diff
@@ -194,21 +194,23 @@ Context::exportOutputs() {
                                                      busesWithDynamicModel_, dynamicDataBaseManager_, dynamicModels_, staticVarCompensators_));
   dydWriter.write();
 
-  // Par
-  // copy constants files
-  for (auto& entry : boost::make_iterator_range(file::directory_iterator(def_.parFileDir))) {
-    if (entry.path().extension() == ".par") {
-      file::path dest(outputDir);
-      dest.append(entry.path().filename().generic_string());
-      file::copy_file(entry.path(), dest, file::copy_option::overwrite_if_exists);
-    }
-  }
   // create specific par
   file::path parOutput(config_.outputDir());
   parOutput.append(basename_ + ".par");
-  outputs::Par parWriter(outputs::Par::ParDefinition(basename_, config_.outputDir(), parOutput, generators_, hvdcLineDefinitions_,
-                                                     config_.getActivePowerCompensation(), busesWithDynamicModel_, dynamicDataBaseManager_, counters_,
-                                                     dynamicModels_, linesById_, staticVarCompensators_, loads_));
+  outputs::Par parWriter(outputs::Par::ParDefinition(basename_,
```

Comment:
as we pass as argument many config_ object attributes, I think it is more readable to pass the reference to config_ object and use it later on. And I'd apply the same logic as much as we have more than 1 config attribute in a method or constructor